README.md

```diff
@@ -70,32 +70,36 @@ environment to a compute cluster.
 
 Once you have [installed Docker](https://docs.docker.com/engine/installation/) you can either use a [prebuilt image from a release](https://hub.docker.com/r/allennlp/models) or build an image locally with any version of `allennlp` and `allennlp-models`.
 
+If you have GPUs available, you also need to install the [nvidia-docker](https://github.com/NVIDIA/nvidia-docker) runtime.
```

Comment:
Are you sure this is true?  It may be, but these things have changed over time so I'm not sure...

Reply:
I'm pretty sure.